### PR TITLE
OCT-62: fix deprecations (with rector and phpstan-deprecation-rules)

### PR DESCRIPTION
**Pull request by masacc**
**Description (for Contributor and Core Developer)**

EE : https://github.com/akeneo/pim-enterprise-dev/pull/17342

- Launch Rector and fix all deprecations in connectivity
- fix all ignored rules in phpstan deprecation rules config
- remove globally unused phpinsights tool
- remove unused psalm config in connectivity

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Connectivity/Connection/back/Application/Apps/Command/RequestAppAuthenticationHandler.php [hunk 1]
```diff
@@ -17,21 +17,8 @@
...
-        GetUserConsentedAuthenticationScopesQueryInterface $getUserConsentedAuthenticationScopesQuery,
-        CreateUserConsentQueryInterface $createUserConsentQuery,
-        ClockInterface $clock,
-        ValidatorInterface $validator
-    ) {
-        $this->getUserConsentedAuthenticationScopesQuery = $getUserConsentedAuthenticationScopesQuery;
-        $this->createUserConsentQuery = $createUserConsentQuery;
-        $this->clock = $clock;
-        $this->validator = $validator;
+    public function __construct(private GetUserConsentedAuthenticationScopesQueryInterface $getUserConsentedAuthenticationScopesQuery, private CreateUserConsentQueryInterface $createUserConsentQuery, private ClockInterface $clock, private ValidatorInterface $validator)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Application/Apps/Command/RequestAppAuthenticationHandler.php:20]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Application/Apps/Service/CreateConnection.php [hunk 2]
```diff
@@ -17,15 +17,8 @@ class CreateConnection implements CreateConnectionInterface
...
-    private ConnectionRepositoryInterface $repository;
-    private SelectConnectionWithCredentialsByCodeQueryInterface $selectConnectionWithCredentialsByCodeQuery;
-
-    public function __construct(
-        ConnectionRepositoryInterface $repository,
-        SelectConnectionWithCredentialsByCodeQueryInterface $selectConnectionWithCredentialsByCodeQuery
-    ) {
-        $this->repository = $repository;
-        $this->selectConnectionWithCredentialsByCodeQuery = $selectConnectionWithCredentialsByCodeQuery;
+    public function __construct(private ConnectionRepositoryInterface $repository, private SelectConnectionWithCredentialsByCodeQueryInterface $selectConnectionWithCredentialsByCodeQuery)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Application/Apps/Service/CreateConnection.php:20]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Application/Settings/Command/UpdateConnectionHandler.php [hunk 3]
```diff
@@ -19,20 +19,8 @@
...
-    public function __construct(
-        ValidatorInterface $validator,
-        ConnectionRepositoryInterface $repository,
-        UpdateUserPermissionsInterface $updateUserPermissions
-    ) {
-        $this->validator = $validator;
-        $this->repository = $repository;
-        $this->updateUserPermissions = $updateUserPermissions;
+    public function __construct(private ValidatorInterface $validator, private ConnectionRepositoryInterface $repository, private UpdateUserPermissionsInterface $updateUserPermissions)
+    {
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Application/Settings/Command/UpdateConnectionHandler.php:23]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Application/Webhook/Query/GetAConnectionWebhookHandler.php [hunk 4]
```diff
@@ -14,18 +14,8 @@
...
-
-    public function __construct(
-        GetAConnectionWebhookQueryInterface $getAConnectionWebhookQuery,
-        int $activeEventSubscriptionsLimit,
-        CountActiveEventSubscriptionsQueryInterface $countActiveEventSubscriptionsQuery
-    ) {
-        $this->getAConnectionWebhookQuery = $getAConnectionWebhookQuery;
-        $this->activeEventSubscriptionsLimit = $activeEventSubscriptionsLimit;
-        $this->countActiveEventSubscriptionsQuery = $countActiveEventSubscriptionsQuery;
+    public function __construct(private GetAConnectionWebhookQueryInterface $getAConnectionWebhookQuery, private int $activeEventSubscriptionsLimit, private CountActiveEventSubscriptionsQueryInterface $countActiveEventSubscriptionsQuery)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Application/Webhook/Query/GetAConnectionWebhookHandler.php:17]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Domain/Audit/Model/Read/PeriodEventCount.php [hunk 5]
```diff
@@ -13,28 +13,11 @@
...
-        string $connectionCode,
-        \DateTimeImmutable $fromDateTime,
-        \DateTimeImmutable $upToDateTime,
-        array $hourlyEventCounts
-    ) {
-        $this->connectionCode = $connectionCode;
-        $this->fromDateTime = $fromDateTime;
-        $this->upToDateTime = $upToDateTime;
-        $this->hourlyEventCounts = $hourlyEventCounts;
+    public function __construct(private string $connectionCode, private \DateTimeImmutable $fromDateTime, private \DateTimeImmutable $upToDateTime, private array $hourlyEventCounts)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Domain/Audit/Model/Read/PeriodEventCount.php:19]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Domain/Marketplace/DTO/GetAllExtensionsResult.php [hunk 6]
```diff
@@ -49,9 +46,7 @@ public function withAnalytics(array $queryParameters): self
     {
         return self::create(
             $this->total,
-            \array_map(function (Extension $extension) use ($queryParameters) {
-                return $extension->withAnalytics($queryParameters);
-            }, $this->extensions),
+            \array_map(fn (Extension $extension): \Akeneo\Connectivity\Connection\Domain\Marketplace\Model\Extension => $extension->withAnalytics($queryParameters), $this->extensions),
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Domain/Marketplace/DTO/GetAllExtensionsResult.php:49]: Can be reduce with a `use`.

File: src/Akeneo/Connectivity/Connection/back/Domain/Settings/Model/Read/Connection.php [hunk 7]
```diff
@@ -13,32 +13,8 @@
...
-        bool $auditable = false,
-        string $type = ConnectionType::DEFAULT_TYPE
-    ) {
-        $this->code = $code;
-        $this->label = $label;
-        $this->flowType = $flowType;
-        $this->image = $image;
-        $this->auditable = $auditable;
-        $this->type = $type;
+    public function __construct(private string $code, private string $label, private string $flowType, private ?string $image = null, private bool $auditable = false, private string $type = ConnectionType::DEFAULT_TYPE)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Domain/Settings/Model/Read/Connection.php:16]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Domain/Settings/Model/Read/ConnectionWithCredentials.php [hunk 8]
```diff
@@ -11,54 +11,10 @@
...
-        $this->flowType = $flowType;
-        $this->clientId = $clientId;
-        $this->secret = $secret;
-        $this->image = $image;
-        $this->username = $username;
-        $this->userRoleId = $userRoleId;
-        $this->userGroupId = $userGroupId;
-        $this->auditable = $auditable;
-        $this->type = $type;
+    public function __construct(private string $code, private string $label, private string $flowType, private ?string $image, private string $clientId, private string $secret, private string $username, private string $userRoleId, private ?string $userGroupId, private bool $auditable, private string $type)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Domain/Settings/Model/Read/ConnectionWithCredentials.php:16]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Domain/Webhook/Model/WebhookEvent.php [hunk 9]
```diff
@@ -14,40 +14,11 @@
...
-        EventInterface $pimEvent
-    ) {
-        $this->action = $action;
-        $this->eventId = $eventId;
-        $this->eventDateTime = $eventDateTime;
-        $this->data = $data;
-        $this->author = $author;
-        $this->pimSource = $pimSource;
-        $this->pimEvent = $pimEvent;
+    public function __construct(private string $action, private string $eventId, private string $eventDateTime, private Author $author, private string $pimSource, private array $data, private EventInterface $pimEvent)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Domain/Webhook/Model/WebhookEvent.php:20]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Apps/OAuth/AuthorizationCodeGenerator.php [hunk 10]
```diff
@@ -19,26 +19,10 @@
...
-        RandomCodeGeneratorInterface $randomCodeGenerator,
-        ClockInterface $clock
-    ) {
-        $this->clientManager = $clientManager;
-        $this->userRepository = $userRepository;
-        $this->storage = $storage;
-        $this->randomCodeGenerator = $randomCodeGenerator;
-        $this->clock = $clock;
+    public function __construct(private ClientManagerInterface $clientManager, private UserRepositoryInterface $userRepository, private IOAuth2GrantCode $storage, private RandomCodeGeneratorInterface $randomCodeGenerator, private ClockInterface $clock)
+    {
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Apps/OAuth/AuthorizationCodeGenerator.php:25]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Apps/User/CreateUserGroup.php [hunk 11]
```diff
@@ -18,21 +18,9 @@
...
-        ObjectUpdaterInterface $userGroupUpdater,
-        SaverInterface $userGroupSaver,
-        ValidatorInterface $validator
-    ) {
-        $this->userGroupFactory = $userGroupFactory;
-        $this->userGroupUpdater = $userGroupUpdater;
-        $this->userGroupSaver = $userGroupSaver;
-        $this->validator = $validator;
+
+    public function __construct(private SimpleFactoryInterface $userGroupFactory, private ObjectUpdaterInterface $userGroupUpdater, private SaverInterface $userGroupSaver, private ValidatorInterface $validator)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Apps/User/CreateUserGroup.php:22]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/ConnectionContext.php [hunk 12]
```diff
@@ -33,14 +27,8 @@ class ConnectionContext implements ConnectionContextInterface
...
-    public function __construct(
-        AreCredentialsValidCombinationQueryInterface $areCredentialsValidCombinationQuery,
-        SelectConnectionCodeByClientIdQueryInterface $selectConnectionCodeByClientIdQuery,
-        ConnectionRepositoryInterface $connectionRepository
-    ) {
-        $this->areCredentialsValidCombinationQuery = $areCredentialsValidCombinationQuery;
-        $this->selectConnectionCodeByClientIdQuery = $selectConnectionCodeByClientIdQuery;
-        $this->connectionRepository = $connectionRepository;
+    public function __construct(private AreCredentialsValidCombinationQueryInterface $areCredentialsValidCombinationQuery, private SelectConnectionCodeByClientIdQueryInterface $selectConnectionCodeByClientIdQuery, private ConnectionRepositoryInterface $connectionRepository)
+    {
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/ConnectionContext.php:31]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/ErrorManagement/Persistence/PurgeConnectionErrorsQuery.php [hunk 13]
```diff
@@ -62,6 +62,9 @@ private function getDeleteAllDocumentsButGivenIdsQuery(array $idsToKeep): array
         ];
     }
 
+    /**
+     * @return array{_source: string[], sort: array<int, array{error_datetime: array{order: string}}>, size: int, query: array{constant_score: array{filter: array{bool: array{filter: array{exists: array{field: string}}[]|array{term: array{connection_code: string}}[], must_not: array{range: array{error_datetime: array{lt: string}}}}}}}}
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/ErrorManagement/Persistence/PurgeConnectionErrorsQuery.php:66]: Terrible to read ^^

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Marketplace/WebMarketplaceApi.php [hunk 14]
```diff
@@ -16,22 +16,10 @@
...
-        WebMarketplaceAliasesInterface $webMarketplaceAliases,
-        LoggerInterface $logger,
-        FeatureFlag $fakeAppsFeatureFlag
-    ) {
-        $this->client = $client;
-        $this->webMarketplaceAliases = $webMarketplaceAliases;
-        $this->logger = $logger;
-        $this->fakeAppsFeatureFlag = $fakeAppsFeatureFlag;
+
+    public function __construct(private ClientInterface $client, private WebMarketplaceAliasesInterface $webMarketplaceAliases, private LoggerInterface $logger, private FeatureFlag $fakeAppsFeatureFlag)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Marketplace/WebMarketplaceApi.php:21]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Service/RandomCodeGenerator.php [hunk 15]
```diff
@@ -26,8 +26,8 @@ public function generate(): string
         }
         // Last resort: mt_rand
         if (empty($randomData)) { // Get 108 bytes of (pseudo-random, insecure) data
-            $randomData = \mt_rand() . \mt_rand() . \mt_rand() . \uniqid((string) \mt_rand(), true) . \microtime(true) .
-                \uniqid((string) \mt_rand(), true);
+            $randomData = \random_int(0, \mt_getrandmax()) . \random_int(0, \mt_getrandmax()) . \random_int(0, \mt_getrandmax()) . \uniqid((string) \random_int(0, \mt_getrandmax()), true) . \microtime(true) .
+                \uniqid((string) \random_int(0, \mt_getrandmax()), true);
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Service/RandomCodeGenerator.php:30]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Service/User/CreateUser.php [hunk 16]
```diff
@@ -18,28 +18,8 @@
...
-        SimpleFactoryInterface $userFactory,
-        ObjectUpdaterInterface $userUpdater,
-        ValidatorInterface $validator,
-        SaverInterface $userSaver
-    ) {
-        $this->userFactory = $userFactory;
-        $this->userUpdater = $userUpdater;
-        $this->validator = $validator;
-        $this->userSaver = $userSaver;
+    public function __construct(private SimpleFactoryInterface $userFactory, private ObjectUpdaterInterface $userUpdater, private ValidatorInterface $validator, private SaverInterface $userSaver)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Service/User/CreateUser.php:21]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/EventSubscribers/EventsApiRequestsLimitEventSubscriber.php [hunk 17]
```diff
@@ -19,27 +19,8 @@
...
-        LimitOfEventsApiRequestsReachedLoggerInterface $limitOfEventsApiRequestsReachedLogger,
-        EventsApiDebugRepositoryInterface $eventsApiDebugRepository
-    ) {
-        $this->getDelayUntilNextRequest = $getDelayUntilNextRequest;
-        $this->webhookRequestsLimit = $webhookRequestsLimit;
-        $this->sleep = $sleep;
-        $this->reachRequestLimitLogger = $reachRequestLimitLogger;
-        $this->limitOfEventsApiRequestsReachedLogger = $limitOfEventsApiRequestsReachedLogger;
-        $this->eventsApiDebugRepository = $eventsApiDebugRepository;
+    public function __construct(private GetDelayUntilNextRequest $getDelayUntilNextRequest, private int $webhookRequestsLimit, private Sleep $sleep, private ReachRequestLimitLogger $reachRequestLimitLogger, private LimitOfEventsApiRequestsReachedLoggerInterface $limitOfEventsApiRequestsReachedLogger, private EventsApiDebugRepositoryInterface $eventsApiDebugRepository)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/EventSubscribers/EventsApiRequestsLimitEventSubscriber.php:22]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/EventsApiDebug/Persistence/PurgeEventsApiSuccessLogsQuery.php [hunk 18]
```diff
@@ -52,6 +52,9 @@ private function getDeleteAllDocumentsButGivenIdsQuery(array $idsToKeep): array
         ];
     }
 
+    /**
+     * @return array{_source: string[], sort: array<int, array{timestamp: array{order: string}}>, size: int, query: array{constant_score: array{filter: array{bool: array{filter: array{exists: array{field: string}}[]|array{terms: array{level: string[]}}[]}}}}}
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/EventsApiDebug/Persistence/PurgeEventsApiSuccessLogsQuery.php:56]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/tests/CatalogBuilder/ConnectedAppLoader.php [hunk 19]
```diff
@@ -25,39 +25,8 @@
...
-        $this->userRoleLoader = $userRoleLoader;
-        $this->OAuthStorage = $OAuthStorage;
-        $this->clientProvider = $clientProvider;
-        $this->createUserGroup = $createUserGroup;
-        $this->createUser = $createUser;
-        $this->createConnection = $createConnection;
-        $this->createApp = $createApp;
-        $this->userRepository = $userRepository;
-        $this->unitOfWorkAndRepositoriesClearer = $unitOfWorkAndRepositoriesClearer;
+    public function __construct(private DbalConnection $dbalConnection, private UserRoleLoader $userRoleLoader, private OAuthStorage $OAuthStorage, private ClientProviderInterface $clientProvider, private CreateUserGroupInterface $createUserGroup, private CreateUserInterface $createUser, private CreateConnectionInterface $createConnection, private CreateConnectedAppInterface $createApp, private UserRepositoryInterface $userRepository, private UnitOfWorkAndRepositoriesClearer $unitOfWorkAndRepositoriesClearer)
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/tests/CatalogBuilder/ConnectedAppLoader.php:28]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/tests/Unit/spec/Infrastructure/Marketplace/Persistence/GetAllExtensionsQuerySpec.php [hunk 20]
```diff
@@ -89,8 +89,6 @@ public function it_execute_and_returns_extension_result(
             ],
         ]);
 
-        $this->execute()->shouldBeLike(GetAllExtensionsResult::create(3, \array_map(function ($item) {
-            return Extension::fromWebMarketplaceValues($item);
-        }, $items)));
+        $this->execute()->shouldBeLike(GetAllExtensionsResult::create(3, \array_map(fn ($item): \Akeneo\Connectivity\Connection\Domain\Marketplace\Model\Extension => Extension::fromWebMarketplaceValues($item), $items)));
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/tests/Unit/spec/Infrastructure/Marketplace/Persistence/GetAllExtensionsQuerySpec.php:92]: Need manual adaptation IMO

File: src/Akeneo/Connectivity/Connection/back/tests/Unit/spec/Infrastructure/Marketplace/Persistence/GetAllAppsQuerySpec.php [hunk 21]
```diff
@@ -171,8 +170,6 @@ public function it_sets_pending_to_true_on_pending_apps(
         $this->items[3]['connected'] = false;
         $this->items[3]['isPending'] = true;
 
-        $this->execute()->shouldBeLike(GetAllAppsResult::create(4, \array_map(function ($item) {
-            return App::fromWebMarketplaceValues($item);
-        }, $this->items)));
+        $this->execute()->shouldBeLike(GetAllAppsResult::create(4, \array_map(fn ($item): \Akeneo\Connectivity\Connection\Domain\Marketplace\Model\App => App::fromWebMarketplaceValues($item), $this->items)));
```
Comment (hogren) [inline src/Akeneo/Connectivity/Connection/back/tests/Unit/spec/Infrastructure/Marketplace/Persistence/GetAllAppsQuerySpec.php:173]: can be reduced with a `use`

**Review by hogren — APPROVED** [review]

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/CustomApps/Persistence/GetAllCustomAppsQuery.php [hunk 22]
```diff
@@ -41,7 +41,7 @@ public function execute(): GetAllCustomAppsResult
 
         return GetAllCustomAppsResult::create(
             \count($rows),
-            \array_map(function ($row) {
+            \array_map(function ($row): \Akeneo\Connectivity\Connection\Domain\Marketplace\Model\App {
```
Comment (guillaumefer-akeneo) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/CustomApps/Persistence/GetAllCustomAppsQuery.php:44]: simplify with a `use`

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/Client/GuzzleWebhookClient.php [hunk 23]
```diff
@@ -84,15 +85,15 @@ public function bulkSend(iterable $webhookRequests): void
...
+                'fulfilled' => function (Response $response, int $index) use (&$logs): void {
                     /** @var EventSubscriptionSendApiEventRequestLog $webhookRequestLog */
                     $webhookRequestLog = $logs[$index];
                     $webhookRequestLog->setSuccess(true);
                     $webhookRequestLog->setEndTime(\microtime(true));
                     $webhookRequestLog->setResponse($response);
 
                     $pimEvents = \array_map(
-                        static fn (WebhookEvent $apiEvent) => $apiEvent->getPimEvent(),
+                        static fn (WebhookEvent $apiEvent): \Akeneo\Platform\Component\EventQueue\EventInterface => $apiEvent->getPimEvent(),
```
Comment (guillaumefer-akeneo) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/Client/GuzzleWebhookClient.php:96]: simplify with a `use`

File: src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Connection/CheckAuthenticationSecurityEndToEnd.php [hunk 24]
```diff
@@ -281,7 +281,7 @@ private function findAConnection(string $connectionCode): ConnectionWithCredenti
     /**
      * @return Configuration
      */
-    protected function getConfiguration()
+    protected function getConfiguration(): \Akeneo\Test\Integration\Configuration
```
Comment (guillaumefer-akeneo) [inline src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Connection/CheckAuthenticationSecurityEndToEnd.php:284]: replace with a `use`

File: src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductEventEndToEnd.php [hunk 25]
```diff
@@ -60,7 +60,7 @@ protected function setUp(): void
         $this->productLoader = static::getContainer()->get(
             'akeneo_connectivity.connection.fixtures.enrichment.product'
         );
-        $this->historyContainer = $this->get('Akeneo\Connectivity\Connection\Tests\EndToEnd\GuzzleJsonHistoryContainer');
+        $this->historyContainer = $this->get(\Akeneo\Connectivity\Connection\Tests\EndToEnd\GuzzleJsonHistoryContainer::class);
```
Comment (guillaumefer-akeneo) [inline src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductEventEndToEnd.php:63]:
```suggestion
        $this->historyContainer = $this->get(GuzzleJsonHistoryContainer::class);
```
+ use

**Review by guillaumefer-akeneo — APPROVED** [review]

File: composer.json
Comment (RiffVentura) [inline composer.json:1]: are we sure other squads don't use it?

**Review by RiffVentura — APPROVED** [review]

**Pull request merged**